src/components/Reports/GoalCalculator/CalculatorSettings/SettingsSectionList.tsx

```diff
@@ -1,14 +1,22 @@
 import React from 'react';
 import { useTranslation } from 'react-i18next';
+import { useGoalCalculator } from '../Shared/GoalCalculatorContext';
+import { getFamilySections } from '../Shared/familySections';
 import { SectionList } from '../SharedComponents/SectionList';
 
 export const SettingsSectionList: React.FC = () => {
   const { t } = useTranslation();
+  const {
+    goalCalculationResult: { data },
+  } = useGoalCalculator();
+  const specialFamily =
+    data?.goalCalculation.specialFamily.primaryBudgetCategories;
 
   const sections = [
     { title: t('Information'), complete: false },
-    { title: t('Special Income'), complete: false },
-    { title: t('One-time Goals'), complete: false },
+    ...(specialFamily
```

Comment:
[Reuse] SettingsStep.tsx has hardcoded sections while this component uses dynamic data. Evidence: SettingsStep:23-28 hardcodes 'Special Income' and 'One-time Goals'. Impact: Inconsistency. Suggestion: Use getFamilySections in SettingsStep too. [Category: Reuse]

Reply:
You're right that `SettingsStep` needs to be changed, but Will is already doing that in his PR: https://github.com/CruGlobal/mpdx-react/pull/1421/files#diff-36afeff2b25aa3047f1c771d61efb8fe34c92da5e19fb8f524ccc90167305852.